start-here.mdx

```diff
@@ -1,8 +1,153 @@
 ---
 title: "Start here"
-icon: rocket
 ---
 
-import { Stub } from '/snippets/stub.jsx';
+import { Aside } from "/snippets/aside.jsx";
 
-<Stub issue="79" />
+This documentation is divided into several sections, with lower-level details appearing later in the list. For example, an existing standard contract may already solve the problem, so there may be no need to implement a new one.
+
+|                        |                                                                                              |
+| ---------------------- | -------------------------------------------------------------------------------------------- |
+| Ecosystem              | Tools and libraries for developing on TON, and how to use them.                              |
+| Payment processing     | Implementation guides for an exchange.                                                       |
+| Standard contracts     | Working with most popular standardized contracts.                                            |
+| Contract development   | Guides on developing smart-contracts.                                                        |
+| Languages              | Reference documentation for TON-specific programming languages.                              |
+| TON Virtual Machine    | Description of the low-level language that runs smart-contracts, and details of the runtime. |
+| Blockchain foundations | Comprehensive description of the blockchain.                                                 |
+| Contribute             | Documentation on writing this documentation.                                                 |
+
+## The minimum
+
+<Aside type="tip">
+  This is a condensed description of TON. The rest of the documentation may assume that all of this is already known to the reader.
+</Aside>
+
+_TON_ is a [blockchain](https://en.wikipedia.org/wiki/Blockchain). It provides a distributed platform for hosting applications. Roughly speaking, it works as if it were a single server executing all the code. Such applications are called _smart-contracts_.
```

Comment:
"distributed platform for hosting applications", then immediately "as if it were a single server executing all the code"

I get the point but it can be misleading for newcomers